config/locales/translations/zendesk_apps_support.yml

```diff
@@ -119,6 +119,14 @@ parts:
       key: "txt.apps.admin.error.app_build.multiple_channel_integrations"
       title: "App builder job: requirements file contains multiple channel integrations, leave requirements.json as is (file name)"
       value: "Specifying multiple channel integrations in requirements.json is not supported."
+  - translation:
+      key: "txt.apps.admin.error.app_build.invalid_cr_schema_keys.one"
+      title: "App builder job: custom resources schema contains invalid key error"
+      value: "Custom resources schema contains an invalid key: %{invalid_types}"
```

Comment:
Can you add screenshots for these 2 keys so that linguists will have better understanding?